prime-router/settings/organizations.yml

```diff
@@ -1067,8 +1067,6 @@
       jurisdictionalFilter:
         - orEquals(ordering_facility_state, NJ, patient_state, NJ)
         - matches(processing_mode_code, P)
-      routingFilter:
-        - matches(test_result_status, F, C, P)
```

Comment:
I added this earlier so you might have to update the branch to include the routing filter in the local org file.